### DISCUSSION TITLE
Is there also support for Body_25 keypoints used by openpose

### DISCUSSION THREAD

**Discussion by sahil-syze** [openpifpaf/openpifpaf#476] [Q&A]
We are exploring pose detection alternatives and I came across your well maintained project. Our internal algorithms take body_25 keypoints as the input and I was wondering if openpifpaf also supports those keypoints?

Reference:
https://cmu-perceptual-computing-lab.github.io/openpose/web/html/doc/md_doc_02_output.html#pose-output-format-body_25

**Comment by svenkreiss** [comment]
Yes, OpenPifPaf is completely flexible with the number of keypoints. The largest models that we have trained so far contain 133 keypoints.

Custom dataset tutorial: https://openpifpaf.github.io/plugins_custom.html
133 keypoints: https://openpifpaf.github.io/plugins_wholebody.html
